pkgs/os-specific/darwin/itsycal/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+, unzip
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "itsycal";
+  version = "0.15.3";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20240326164201/https://itsycal.s3.amazonaws.com/Itsycal.zip";
+    hash = "sha256-5aJzSuqq31B33jW4lV8vuU3eurpZBoyIW/AOC9/pxng=";
+  };
+
+  sourceRoot = ".";
+
+  nativeBuildInputs = [ unzip ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    mv Itsycal.app $out/Applications
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Itsycal is a tiny menu bar calendar";
+    homepage = "https://www.mowglii.com/itsycal/";
+    license = lib.licenses.free;
```

Comment:
I can't find any explicit licensing statement on their website, probably we should use `unfree` here?

Reply:
I think you're looking at the old commit as it's `free` on commit 52269f864923947582e230028dbec096cafdee78 although I will change to mit

Reply:
You are right! It's MIT: https://github.com/sfsam/Itsycal/blob/master/LICENSE.txt